.github/workflows/gh-pages.yml

```
@@ -0,0 +1,96 @@
+name: Publish Web Bundle
+
+on:
+  # Our web bundle is published manually now,
+  # but this trigger is going to be changed to an event that guideline has been updated
+  push:
+    branches:
+    - publish-webbundle
+
+jobs:
+  build-publish:
+    runs-on: ubuntu-18.04
+    env:
+      TZ: Asia/Tokyo
+      BUNDLE_NAME: ameba-accessibility-guidelines.wbn
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        ref: ${{ github.ref }}
+
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+
+    - name: Setup Hugo
+      uses: peaceiris/actions-hugo@v2.3.0
+      with:
+        hugo-version: 0.55.6
+
+    - name: Build CSS
+      run: |
+        npm install
+        npm run build
+    
+    - name: Build Hugo
+      run: hugo --minify
+
+    - name: Install go/bundle CLI
+      run: |
+        # Go has been installed at the setup hugo step,
+        # you can run setup-go action if you want to install specific version
+        go get -u github.com/WICG/webpackage/go/bundle/cmd/...
+        # Add path to run the package
+        echo "::add-path::$HOME/go/bin"
+    
+    - name: Generate Web Bundle 📦
+      run: |
+        # Create URL list from the files built by Hugo
+        #   - directory index: replacing /index.html with ''
+        #     - URL should be exactly the same with the link text in the document
+        #   - remove empty lines
+        #   - add hosting domain
+        find public -type f | sed -e 's/\/index.html//g' -e '/^$/d' -e 's/public/https:\/\/openameba.github.io\/a11y-guidelines/g' > urls.txt
```

sedの`s`は次の文字を区切り文字にするので`/`以外の文字で書いてあげた方が読みやすいかな〜と思いました

```suggestion
        find public -type f | sed -e 's|/index.html||g' -e '/^$/d' -e 's|public|https://openameba.github.io/a11y-guidelines|g' > urls.txt
```

---

findも末尾に`-print`つけた方がいいんだろうか……🤔（動作が微妙に変わる気がするんですが`-type f`つけてるから変わらないのかな）